src/main/java/WayofTime/bloodmagic/ritual/types/RitualPump.java

```diff
@@ -81,6 +81,7 @@ else if (state.getBlock() instanceof IFluidHandler)
                 fluidHandler.fill(posInfo.getRight(), true);
                 world.setBlockState(posInfo.getLeft(), Blocks.STONE.getDefaultState());
                 world.notifyBlockUpdate(posInfo.getLeft(), tankState, tankState, 3);
```

Comment:
I believe this should pass `Blocks.STONE.getDefaultState()` for both parameters. Since you're in here, could you make that change?

Reply:
Done